website_sale_cache/README.rst

```diff
@@ -2,20 +2,22 @@
 Website sale cache
 ==================
 
-This module caches product public categories from module "website_sale". 
-That allows to significantly accelerate the loading of the page with a large number of product categories.
+The module allows to cache product website categories and significantly accelerate the page loading with a large number
+of product categories.
 
-Note
-----
-When used with multiple odoo workers, the cache is updated separately for each one. So the page may be load slowly until
-the cache is updated for each worker. After the cache is updated for each worker, the page will load faster, as far as
-the advantages of caching allow.
+When multiple odoo workers are used, the cache is updated for each one separately. So the page may be loaded slowly
+before updating the cache for each worker. Once the latter is done, the page will be loading faster,
+as far as the advantages of caching allow.
+
+The speed increase will be absent at the first load, because the data has not yet been cached. Also when updating
+the list of categories (deleting a category, updating or creating a new one), the cache will be updated, which will also
+cause an absent of speed increase at the first load after making changes.
```

Comment:
The speed is not increased at the first load, because the data has not been cached yet. After update the list of categories (deleting a category, updating or creating a new one), the cache is updated and once again the speed will increase only after the first load.